addons/forum-post-save/addon.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Save draft posts",
+  "description": "Saves posts that you haven't sent yet at an interval.",
```

Here's my description:
```suggestion
  "description": "Saves forum posts that you haven't submitted.",
```